### PR TITLE
Create 2015-06-23-make-subplots-shared-yaxes

### PR DESCRIPTION
**Pull request by aneda**
@chriddyp should I get rid of  the title

### CODE REVIEW

File: _posts/python/subplots/2015-06-23-make-subplots-shared-yaxes [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+fig = tools.make_subplots(rows=2, cols=2, shared_yaxes=True)
+
+fig.append_trace(trace1, 1, 1)
+fig.append_trace(trace2, 1, 2)
+fig.append_trace(trace3, 2, 1)
+fig.append_trace(trace4, 2, 2)
+
+fig['layout'].update(height=600, width=600, title='Multipule Subplots with Shared Y-axes')
+plot_url = py.plot(fig, filename='multipule-subplots-shared-yaxes')
```
Comment (chriddyp) [inline _posts/python/subplots/2015-06-23-make-subplots-shared-yaxes]: @aneda - multiple spelled wrong here and in the title

File: _posts/python/subplots/2015-06-23-make-subplots-shared-yaxes [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+fig = tools.make_subplots(rows=2, cols=2, shared_yaxes=True)
+
+fig.append_trace(trace1, 1, 1)
+fig.append_trace(trace2, 1, 2)
+fig.append_trace(trace3, 2, 1)
+fig.append_trace(trace4, 2, 2)
+
+fig['layout'].update(height=600, width=600, title='Multipule Subplots with Shared Y-axes')
```
Comment (chriddyp) [inline _posts/python/subplots/2015-06-23-make-subplots-shared-yaxes]: @aneda - use title case: http://titlecapitalization.com, update "Y-axes" to "Y-Axes"

File: _posts/python/subplots/2015-06-23-make-subplots-shared-yaxes [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+name: Subplots with Shared Y-axes
```
Comment (chriddyp) [inline _posts/python/subplots/2015-06-23-make-subplots-shared-yaxes]: use title case: http://titlecapitalization.com, Y-axes -> "Y-Axes"

**Comment by aneda** [comment]
@chriddyp thanks for http://titlecapitalization.com/  it's awesome!  should I merge?

**Comment by chriddyp** [comment]
looks good!

**Pull request merged**